src/AI/Attack.pm

```diff
@@ -44,6 +44,24 @@ sub process {
 		delete $args->{suspended};
 	}
 
+	if ($args->{ID}) {
+		my $target = Actor::get($args->{ID});
+		if ($target) {
+			my $target_is_aggressive = is_aggressive($target);
+			my @aggressives = ai_getAggressives();
+			if ($config{attackChangeTarget} && !$target_is_aggressive && @aggressives) {
+				$char->sendAttackStop;
+				AI::dequeue;
+				AI::dequeue if (AI::action eq "route");
+				AI::dequeue if (AI::action eq "attack");
+				my $attackTarget = getBestTarget(\@aggressives, $config{attackCheckLOS}, $config{attackCanSnipe});
+				ai_setSuspend(0);
+				warning TF("[attackChangeTarget] %s, target %s is not aggressive, changing target to aggressive %s.\n", $char, $target, $attackTarget), 'ai_attack';
```

Comment:
Does this feature only apply to the character? Can a mercenary or homunculus use this function?
I think this line should be replaced with:
```
warning TF("Your target is not aggressive: %s, changing target to aggressive: %s.\n", $target, $attackTarget), 'ai_attack';
```

Reply:
Using this PR yes, only the character can use it, when I get home I will review this a little more and add the homunculus functionality, until then lets wait to merge this